dxcint/dxcint/Terraform.py

```diff
@@ -98,33 +98,40 @@ def _make_prerequisites(self, language: str) -> str:
 
     @async_retry()
     def _build_asset(self, language: str) -> str:
+
+        def find_asset_obj(self: Terraform, asset_name: str, zero_ok: bool) -> str:
+            return dxpy.search.find_one_data_object(
+                classname="record",
+                project=self._context.project_id,
+                name=asset_name,
+                folder=self._context.platform_build_dir,
+                more_ok=False,
+                zero_ok=zero_ok
+            )
+
```

Comment:
I would recommend avoiding nested declaration of functions. - the less nested code you have the better it is for the readability. If you do that to keep the interface of a nested function simpler (i.e. by using already declared vars in the outer scope) - you risk to lose a track of the state of those vars during the execution of the outer function. 
Since we don't need `zero_ok` (see my other comment) - I would revert.